p2p/simulations/adapters/exec.go

```diff
@@ -107,7 +110,10 @@ func (e *ExecAdapter) NewNode(config *NodeConfig) (Node, error) {
 
 	// listen on a random localhost port (we'll get the actual port after
 	// starting the node through the RPC admin.nodeInfo method)
-	conf.Stack.P2P.ListenAddr = "127.0.0.1:0"
```

Comment:
Ultimately I don't think we can do that, because we need to know the concrete `ListenAddr`, so that we setup the Bzz protocol with the correct `addr.Under()` underlay address.